<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="btn8">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" rowspan="2">
    <widget class="QPushButton" name="btndengyu">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>180</height>
      </size>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn4">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnclear">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnchu">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btn9">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="btnjian">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btn2">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn5">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnshan">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btncheng">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btn1">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnmo">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btn7">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btn3">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" rowspan="2">
    <widget class="QPushButton" name="btn0">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnjia">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btn6">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QLabel" name="result">
     <property name="text">
      <string>输出结果</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
